### DISCUSSION TITLE
Add compatibility for Firefox for Android (Fenix)

### DISCUSSION THREAD

**Discussion by ph00lt0** [simple-login/app#1281] [Feature Requests]
Please add support for the android version of Firefox. Would be useful to have generation of email addresses also on request there.

**Comment by ph00lt0** [comment]
Not sure why this got transferred to the app repo? This surely is related to the browser add-on @nguyenkims.

**Reply by nguyenkims** [reply]
Hey we centralise all feature requests on this forum so people can upvote it. This forum happens to be part of the app repo but it contains discussions related to all SimpleLogin apps (web app, mobile app, extension)

**Reply by ph00lt0** [reply]
Right, I would suggest making a separate repo for discussions then because this is rather confusing.
